clients/views.py

```diff
@@ -79,12 +76,10 @@ def get(self, *args, **kwargs):
 		return render(self.request, self.template_name, context=self.get_context_data(**kwargs))
 
 	def post(self, *args, **kwargs):
-		#import pdb; pdb.set_trace()
-		client_form = ClientForm(self.request.POST,user=self.request.user)	
-		#import pdb; pdb.set_trace()
+		client_form = ClientForm(self.request.POST,user=self.request.user )	
 		if client_form.is_valid() :
 			
-			client = client_form.save(commit=False)		
+			client = client_form.save(commit=False,company=self.request.user.company)		
 			
 			client.save()
 			messages.success(self.request, 'Client is successfully added')
```

Comment:
avoid using this module. messages module.

learn to use status code to define the action. then assign the text message in the front-end instead